test/OZ/ownable/CrowdFund.js

```diff
@@ -0,0 +1,188 @@
+    ownerAddress = await signers[0].getAddress()
+
+    const crowdFundFactory = await ethers.getContractFactory('CrowdFund')
+
+    crowdFund = await crowdFundFactory.deploy(ownerAddress)
+  })
+
+  it('Deployment', async () => {
+    const contractOwner = await crowdFund.owner()
+    const contractBalance = await crowdFund.balance()
+
+    expect(contractBalance).to.eq(0)
+    expect(contractOwner).to.eq(ownerAddress)
+    expect(ethers.utils.isAddress(crowdFund.address)).to.be.true
+  })
+
+  it('Should deposit an amount of HBAR', async () => {
+    // prepare funder
+    const funder = signers[1]
+
+    // prepare transaction
+    const tx = await crowdFund.connect(funder).deposit({ value: FUND_AMOUNT })
+
+    // wait for receipt
+    const receipt = await tx.wait()
+
+    // extract event arguments
+    const [funderAddress, fundedAmount] = receipt.events.map(
+      (e) => e.event === 'Deposit' && e
+    )[0].args
+
+    // retrieve contract balance
+    const contractBalance = await crowdFund.balance()
+
+    // assertions
+    expect(funderAddress).to.eq(await funder.getAddress())
+    expect(fundedAmount).to.eq(Math.round(FUND_AMOUNT / TINY_BAR_TO_WEI_COEF))
+    expect(contractBalance).to.eq(
+      Math.round(FUND_AMOUNT / TINY_BAR_TO_WEI_COEF)
+    )
+  })
+
+  it('Should allow owner to withdraw an amount which is less than contract balance', async () => {
+    // prepare signers
+    const owner = signers[0]
+    const funder = signers[1]
+
+    // fund the contract by the funder
+    await crowdFund.connect(funder).deposit({ value: FUND_AMOUNT })
+
+    // prepare transaction to withdraw an amount by owner
+    const WITHDRAWN_AMOUNT = 10000000000
+    const tx = await crowdFund
+      .connect(owner)
+      .withdraw(WITHDRAWN_AMOUNT / TINY_BAR_TO_WEI_COEF)
+
+    // wait for receipt
+    const receipt = await tx.wait()
+
+    // extract event arguments
+    const [ownerAddress, withdrawnAmount] = receipt.events.map(
+      (e) => e.event === 'Withdraw' && e
+    )[0].args
+
+    // retrieve contract balance
+    const contractBalance = await crowdFund.balance()
+
+    // assertions
+    expect(ownerAddress).to.eq(await owner.getAddress())
+    expect(withdrawnAmount).to.eq(
+      Math.floor(WITHDRAWN_AMOUNT / TINY_BAR_TO_WEI_COEF)
+    )
+    expect(contractBalance).to.eq(
+      Math.floor((FUND_AMOUNT - WITHDRAWN_AMOUNT) / TINY_BAR_TO_WEI_COEF)
+    )
+  })
+
+  it('Should NOT allow owner to withdraw an amount which is greater than the contract balance', async () => {
+    // prepare signers
+    const owner = signers[0]
+    const funder = signers[1]
+
+    // fund the contract by the funder
+    await crowdFund.connect(funder).deposit({ value: FUND_AMOUNT })
+
+    // prepare transaction to withdraw an amount by owner
+    const WITHDRAWN_AMOUNT = 40000000000 // > FUND_AMOUNT
+    const tx = await crowdFund
+      .connect(owner)
+      .withdraw(WITHDRAWN_AMOUNT / TINY_BAR_TO_WEI_COEF)
+
+    try {
+      await tx.wait()
```

Comment:
Add an `expect` to ensure there was actually an error thrown.

Reply:
Isn't the line 126 expect the error to be thrown? Sorry I'm a little confused by this.

Reply:
Line 126 is fine but what if like 124 doesn't result in an error. 
This test will still pass.
We're expecting an error, if one isn't throw we should fail.
Either insert an expect after line 124 or one outside of the try/catch

Reply:
Ah yeah you're right that makes sense! Thanks for pointing it out!